build/write_bandreps_from_bandrepset.jl

```diff
@@ -50,11 +50,11 @@ function bandrep2csv(io::IO, BRS::BandRepSet)
 end
 
 # Load ::BandRepSet for all 2D plane groups
-include("tbd-filename.jl") # should define BRS_vec
+include("Band_Representations.jl") # should define BRS_vec
 
 # for allpaths in (false, true)
 # for elementary in ...
-for (sgnum, BRS) in enumerate(BRS_vec)
-    filename = "../data/2d/elementary/allpaths/$sgnum.csv"
+for (bandrepset2Dindex, BRS) in zip(BRS_NUMS, BRS_VEC)
+    filename = "../data/bandreps/2d/elementary/allpaths/$bandrepset2Dindex.csv"
```

Comment:
I think what you're writing is actually maxpaths rather than allpaths? Would be nice to write both.
(So the file-location for the new .csv files should be `.../2d/elementary/maxpaths/...` rather than `.../2d/elementary/allpaths/...`)